action.yml

```diff
@@ -23,6 +23,12 @@ inputs:
     description: "Max amount of time to give the server to reload if it needs to"
     required: false
     # no default to let the custom Scenario timeouts create the reload timeout
+  # INSTALL_LOCATION? Will that get confused with the location passed into the command line?
+  # Maybe `INSTALL_LOCATION` could be "playground", "server", or a custom path and we could handle all those.
+  INSTALL_METHOD:
```

Comment:
`INSTALL_METHOD` was an idea that didn't come up when we were discussing today; I like it! I'd be fine with that or `INSTALL_TYPE`.